go/host/p2p/metrics_util.go

```diff
@@ -0,0 +1,39 @@
+package p2p
+
+import gethmetrics "github.com/ethereum/go-ethereum/metrics"
+
+// perStringGaugeMap is a map that returns metrics per string
+// errors per host or events per type are common usages
+type perStringGaugeMap struct {
```

Comment:
let's call this "perHostMetrics" to make its function explicit. 
I was a bit confused and had to look deeper to understand what's going on.

Reply:
Yeah, much better!